envoy/ratelimit/ratelimit.h

```diff
@@ -30,14 +30,15 @@ struct RateLimitOverride {
 struct DescriptorEntry {
   std::string key_;
   std::string value_;
+  uint32_t hits_addend_;
 
   friend bool operator==(const DescriptorEntry& lhs, const DescriptorEntry& rhs) {
-    return lhs.key_ == rhs.key_ && lhs.value_ == rhs.value_;
+    return lhs.key_ == rhs.key_ && lhs.value_ == rhs.value_ && lhs.hits_addend_ == rhs.hits_addend_;
   }
   template <typename H>
   friend H AbslHashValue(H h, // NOLINT(readability-identifier-naming)
                          const DescriptorEntry& entry) {
-    return H::combine(std::move(h), entry.key_, entry.value_);
+    return H::combine(std::move(h), entry.key_, entry.value_, entry.hits_addend_);
```

Comment:
I think `hits_addend` shouldn't be used to compute the hash or be used to compare the entry.